mindmeld/markup.py

```diff
@@ -260,7 +259,9 @@ def process_markup(markup, query_factory, query_options):
     try:
         raw_text, annotations = _parse_tokens(_tokenize_markup(markup))
         query = query_factory.create_query(raw_text, **query_options)
-        entities = _process_annotations(query, annotations)
+        entities = _process_annotations(
+            query, annotations, sys_resolver=query_factory.sys_recognizer
```

Comment:
can we rename from `sys_recognizer ` to `system_entity_recognizer ` everywhere? I dont think `sys_recognizer` is clear in it's meaning.

Reply:
done